source/_integrations/bluetooth.markdown

```diff
@@ -35,6 +35,16 @@ For Bluetooth to function on Linux systems, the [D-Bus](https://en.wikipedia.org
 - Home Assistant Supervised: The host system must run BlueZ, and the D-Bus socket must be accessible to Home Assistant **inside** the container.
 - Home Assistant Core: The system must run BlueZ, and the D-Bus socket must be accessible to Home Assistant.
 
+## Additional setup for Container and Supervised
+
+### Making the DBus socket available in the docker container
+
+For most systems, the Dbus socket is in `/run/dbus`. The socket must be available in the container for Home Assistant to be able to connect to Dbus and access the Bluetooth adapter. When starting with `docker run`, this can be accomplished by adding `-v /run/dbus:/run/dbus:ro` to the command. If the Dbus socket is in `/var/run/dbus` on the host system, use `-v /var/run/dbus:/run/dbus:ro` instead.
+
+### Installing BlueZ
+
+On debian based host systems, the `sudo apt-get -y install bluez` command will install BlueZ.
+
```

Comment:
```suggestion
{% details Instructions %}

### Making the DBus socket available in the docker container

For most systems, the Dbus socket is in `/run/dbus`. The socket must be available in the container for Home Assistant to be able to connect to Dbus and access the Bluetooth adapter. When starting with `docker run`, this can be accomplished by adding `-v /run/dbus:/run/dbus:ro` to the command. If the Dbus socket is in `/var/run/dbus` on the host system, use `-v /var/run/dbus:/run/dbus:ro` instead.

### Installing BlueZ

On debian based host systems, the `sudo apt-get -y install bluez` command will install BlueZ.

{% enddetails %}
```